docs/source/specs/typespec/main.tsp

```diff
@@ -140,9 +140,52 @@ namespace Workspaces {
         statusCode: 204,
     }
 
+    model PaginationMeta {
```

Comment:
The models `PaginationMeta` and `PaginationLinks` are inside the `Workspaces` namespace, maybe we can create a namespace called `Pagination` for them and let these models be used for other namespaces, operations, etc.

Reply:
@vbelchio really struggling to find the "correct"/working way to share models between versioned namespaces. The documentation doesn't seem to provide much help. I'm not sure if it's a side-effect of the way we have things configured, or it's just difficult to do in general. We may need to go with this as is for now and come back to it when we want to abstract it out for other models?